osx/KA-Lite-Monitor/KA-Lite-Monitor.xcodeproj/project.pbxproj

```diff
@@ -327,7 +327,7 @@
 				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
-				CODE_SIGN_IDENTITY = "Foundation for Learning Equality, Inc.";
+				CODE_SIGN_IDENTITY = "";
```

Comment:
Same here. Use the full common name, a.k.a. `Developer ID Application: Foundation for Learning Equality, Inc. (H83B64B6AV)`